### PR TITLE
Fix spp warning in windows

### LINKED ISSUE #15527: spp/p/spp.h(90,11): warning: implicit declaration of function 'getpid'
```
[517/1302] Compiling C object shlr/cafa177@@r2spp@sta/spp_spp.c.obj.
In file included from ../shlr/spp/spp.c:5:
In file included from ..\\shlr\\spp/config.h:19:
..\\shlr\\spp/p/spp.h(90,11): warning: implicit declaration of function 'getpid' is invalid in C99 [-Wimplicit-function-declaration]
        srandom (getpid ()); // TODO: change this to be portable
                 ^
1 warning generated.
```

### PR DESCRIPTION
**Pull request by dav1901**
Fix #15527.

### CODE REVIEW

File: shlr/spp/r_api.h [hunk 1]
```diff
@@ -22,6 +22,7 @@ void r_strbuf_fini(SStrBuf *sb);
 void r_strbuf_init(SStrBuf *sb);
 int r_sys_setenv(const char *key, const char *value);
 char *r_sys_getenv(const char *key);
+unsigned int r_sys_getpid();
```
Comment (radare) [inline shlr/spp/r_api.h]: pid is pid_t, and on unix, pid_t have sign, `typedef int pid_t`

Comment (radare) [reply shlr/spp/r_api.h]: also, if u want to use unsigned int, use `ut32` or `ut64`

**Comment by radare** [comment]
also, this PR should be done in the SPP repo, not here

**Comment by dav1901** [comment]
> also, this PR should be done in the SPP repo, not here

https://github.com/radare/spp/pull/26

**Comment by radare** [comment]
Now reopen this pr

**Pull request merged**